pkgs/tools/wayland/wdomirror/default.nix

```diff
@@ -20,18 +21,19 @@ stdenv.mkDerivation {
     sha256 = "1fz0sajhdjqas3l6mpik8w1k15wbv65hgh9r9vdgfqvw5l6cx7jv";
   };
 
-  nativeBuildInputs = [ meson ninja pkg-config wayland-protocols ];
-
-  buildInputs = [ wayland ];
-
   patches = [
     # https://github.com/progandy/wdomirror/pull/7
-    (fetchurl {
+    (fetchpatch {
       url = "https://github.com/progandy/wdomirror/commit/142632208e9ea2b4a4ebd784532efdb8cad7b87c.patch";
       hash = "sha256-z6/8q2vOtmlGFbDVG5BVWWbLQT8kBvJXT9/oZkWS9gw=";
```

Comment:
Does this really not change the hash? (IIRC `fetchpatch` does quite some normalization.)

Reply:
Yeah, I do get `sha256-MG71IEwRAjjacAkRoB7Tn45+FbY7LAqTDkVJkoWuQUU=` as new hash.

Reply:
changed it, thanks